CSharp8/DefaultInterfaceMethods/BasicDemo.cs

```diff
@@ -21,8 +21,10 @@ interface IFoo
 
             void CallMTwice()
             {
+                Console.WriteLine("Calling M twice in Foo1");
```

Comment:
It would be better to say "Calling M twice in default interface implementation". Logically IFoo shouldn't know or care about Foo1.